packages/world-module-erc20/package.json

```diff
@@ -10,17 +10,17 @@
   "license": "MIT",
   "type": "module",
   "exports": {
-    "./mud.config": "./dist/mud.config.js",
-    ".": "./dist/index.js",
-    "./out/*": "./out/*"
+    "internal": "./dist/internal.js",
+    "mud.config": "./dist/mud.config.js",
+    "out/*": "./out/*"
```

Comment:
ooc why did these change away from `./` prefix? iirc this had some weird downstream consequences

Reply:
reverting, I must have gotten confused, thanks for catching that